e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -219,4 +220,35 @@ describe('Customization', () => {
             expect(response.body.SupportSettings.AboutLink).to.equal(newAboutLink);
         });
     });
+
+    it('SC20329 - Can change Terms of Service Link setting', () => {
+        // * Verify site name's setting name for is visible and matches the text
+        cy.findByTestId('SupportSettings.TermsOfServiceLinklabel').scrollIntoView().should('be.visible').and('have.text', 'Terms of Service Link:');
+
+        // * Verify the site name input box has default value. The default value depends on the setup before running the test.
+        cy.findByTestId('SupportSettings.TermsOfServiceLinkinput').should('have.value', origConfig.SupportSettings.TermsOfServiceLink);
+
+        // * Verify the site name's help text is visible and matches the text
+        cy.findByTestId('SupportSettings.TermsOfServiceLinkhelp-text').find('span').should('be.visible').and('have.text',
+            'Link to the terms under which users may use your online service. By default, this includes the ' +
+            '"Mattermost Conditions of Use (End Users)" explaining the terms under which Mattermost software is ' +
+            'provided to end users. If you change the default link to add your own terms for using the service you ' +
+            'provide, your new terms must include a link to the default terms so end users are aware of the Mattermost ' +
+            'Conditions of Use (End User) for Mattermost software.');
+
+        // # Enter a new help link
+        const newValue = 'https://test.com';
+        cy.findByTestId('SupportSettings.TermsOfServiceLinkinput').clear().type(newValue);
+
+        // # Click Save button
```

Comment:
What's the meaning of the `*` and `#` in these comments?

Reply:
`*` indicates verification/assertion and `#` as test steps. It's a style guide [normally found on header of each spec file](https://github.com/mattermost/mattermost-webapp/pull/4370/files#diff-e8e5668a9849acec0e1a98afb29a0883L4-L8) that helps non-tech QA understand the test flow without digging the codes and be able to compare whether the written code represents the written test case.

Reply:
That makes sense. I haven't seen those headers at the tops of files. I'll make sure to get people to add those when reviewing E2E tests in the future